src/lib/utils/promotions.js

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment'
+import config from '../../config/config'
+
+const { cryptoLiteracyNovemberEndDate } = config
+
+export const isCryptoLiteracyNovember = () => moment().isSameOrBefore(cryptoLiteracyNovemberEndDate, 'day')
```

Comment:
remove this, calculate it directly in src/config/config.js
name it as isCryptoLiteracy